code/stable-website/index.html

```diff
@@ -136,7 +136,7 @@ <h1>About</h1>
           As described in the <a href="https://github.com/JacquesCarette/Drasil/blob/master/Papers/GOOL/GOOL.pdf">GOOL</a> paper, this is a Generic Object-Oriented Language that provides intermediary assistance in code generation, allowing Drasil to more efficiently generate code in several languages, including Python, Java, C-Sharp, and C++.
         </p>
         <p class="paragraph">
-          A list of papers and documents written about Drasil can be found in the <a href="https://github.com/JacquesCarette/Drasil/wiki/Drasil-Papers-and-Documents">Drasil Papers and Documents</a> wiki page
+          A list of papers and documents written about Drasil can be found in the <a href="https://github.com/JacquesCarette/Drasil/wiki/Drasil-Papers-and-Documents">Drasil Papers and Documents</a> wiki page. In particular, there is a <a href="https://github.com/JacquesCarette/Drasil/blob/master/People/Dan/ICSE%20Workshop%20-%20SE4Science/ICSE_LiterateFrameworkForSCSoftware_LSS.pdf">Old Position Paper</a> outlining our original ideas,  a <a href="https://github.com/JacquesCarette/Drasil/blob/master/People/Dan/CAS%20Poster%20Competition/Poster/DrasilPoster.pdf">Drasil Poster</a>, and  a <a href="https://github.com/JacquesCarette/Drasil/blob/master/Papers/WellUnderstood/wu.pdf">Well-Understood Paper</a> discussing key concepts.
```

Comment:
It should be "an" instead of "a" before "Old Position Paper", and it looks like there are extra spaces before the later items (i.e., "a Drasil Poster" and "a Well-Understood Paper").

Also, should "paper" and "poster" be capitalized in these items? My hunch is "no".